### PR TITLE
Support multiple short options handling

### PR DESCRIPTION
**Pull request by YuryBandarchuk16**

### CODE REVIEW

File: tests/OptionParsingTest.hh [hunk 1]
```diff
@@ -118,4 +118,9 @@ final class OptionParsingTest extends TestCase {
     list($cli, $_, $_) = self::cli('--string=--');
     expect($cli->stringValue)->toBeSame('--');
   }
+
+  public function testMultipleShortOptionsWithAssignment(): void {
+    $this->expectException(InvalidArgumentException::class);
+    self::cli('-vs=something');
```
Comment (fredemmott) [inline tests/OptionParsingTest.hh]: This mostly looks good - can you add a non-exception test case please?

Comment (YuryBandarchuk16) [reply tests/OptionParsingTest.hh]: @fredemmott Done

Comment (fredemmott) [reply tests/OptionParsingTest.hh]: https://github.com/facebook/hhvm/commit/21e935844cc490d740009c87ce7707a971a1e8e0#diff-84dbfe1742ea02268109f69440abdd8a fixed them

**Comment by fredemmott** [comment]
Keep under 80 cols in the future please (hackfmt will deal with this, and I'll fix it up)

**Pull request merged**